app/you/edit-profile/index.tsx

```diff
@@ -0,0 +1,288 @@
+import * as ImagePicker from 'expo-image-picker';
+import { router, useNavigation } from 'expo-router';
+import React, { useContext, useEffect, useLayoutEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { BackHandler, TouchableOpacity, View } from 'react-native';
+
+import PictureUpdate from '@/components/modules/userdetails/PictureUpdate';
+import Box from '@/components/ui/box/Box';
+import CustomButton from '@/components/ui/button/CustomButton';
+import CustomHeader from '@/components/ui/header/CustomHeader';
+import Loader from '@/components/ui/loader/Loader';
+import MultiTextInput from '@/components/ui/text-input/MultiTextInput';
+import NumericTextInput from '@/components/ui/text-input/NumericTextInput';
+import SingleTextInput from '@/components/ui/text-input/SingleTextInput';
+import SelectInput from '@/components/ui/text-input/select/SelectInput';
+import { GlobalContext } from '@/context/GlobalContext';
+import { ProfileUpdateContext } from '@/context/userdetails/ProfileUpdateContext';
+import { SelectInputData } from '@/context/utils/SelectInputContext';
+import useProfilePicture from '@/hooks/attachment/UseProfilePicture';
+import useUpdateProfilePicture from '@/hooks/attachment/UseUpdateProfilePicture';
+import useUpdateUserDetails from '@/hooks/userdetails/UseUpdateUserDetails';
+import useUserDetails, { PaymentMethod } from '@/hooks/userdetails/UseUserDetails';
+import { handleImageChoice } from '@/util/HandleImageChoice';
+
+export default function Index() {
+  const { t } = useTranslation();
+  const { userData } = useContext(GlobalContext);
+  const navigation = useNavigation();
+
+  const { authState } = useContext(GlobalContext);
+  const { data: userDetails, isFetching: isFetchingUserDetails } = useUserDetails();
+  const { data: profilePicture, isFetching: isFetchProfilePicture } = useProfilePicture(
+    authState.userId!,
```

Comment:
why we have use that operator afer the variable?

Reply:
case if we pass in that place undefined

Reply:
removed !